.github/workflows/chart-release.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
       -
         name: Package Helm Chart
         run: |
-          sed -i "s/VERSION/${{ github.ref_name }}/g" ./${{ env.CHART_NAME }}/chart/Chart.yaml
+          sed -i -e '1 s/version: v[0-9]*\.[0-9]*\.[0-9]*/version: ${{ github.ref_name }}/; t' -e '1,// s//version: ${{ github.ref_name }}/' ./${{ env.CHART_NAME }}/chart/Chart.yaml
```

Comment:
I find this pretty impenetrable. Is `github.ref_name` a tag? a SHA? either?

Reply:
its the tag. So when you create a new release and give it a tag, this name is used to generate the name for you. Its overly complicated for two reasons.

1) In case there are multiple "version" variables in the Chart.yaml, like if we add a dependency to fleet-scheduler. We only want to grab the first one

2) There is a simpler method of doing "only the first entry", but its GNU only, so just to be sure, i wanted to make sure it was using standard `sed` stuff.